examples/cli/hello_world.py

```diff
@@ -10,7 +10,7 @@
 
 import torch
 
-torch.manual_seed(0)
+torch.manual_seed(1)
```

Comment:
why?

Reply:
So it doesn’t have an identical hash with TwoModels